redis/deprecated/redisbp/monitored_client.go

```diff
@@ -2,102 +2,40 @@ package redisbp
 
 import (
 	"context"
-	"io"
 	"time"
 
-	"github.com/go-redis/redis/v7"
+	"github.com/go-redis/redis/v8"
 	"github.com/reddit/baseplate.go/metricsbp"
 )
 
-// MonitoredCmdable wraps the redis.Cmdable interface and adds additional methods
-// to support integrating with Baseplate.go SpanHooks.
-type MonitoredCmdable interface {
-	// Close should generally not be called directly on a MonitoredCmdable, since
-	// they are meant to be shared and long lived.  It will be called by
-	// MonitoredCmdableFactory.Close which should be called when a server is shut
-	// down.
-	io.Closer
-	redis.Cmdable
-
+// PooledStatser is an interface with PoolStats that reports pool related metrics
+type PooledStatser interface {
```

Comment:
it should be `PoolStatser`, not `PooledStatser`?

Reply:
also I'm still not very sure how this is used. searching for `PooledStatser` in this diff and I don't see it used anywhere else?

Reply:
https://github.com/reddit/baseplate.go/pull/381/files#diff-7463d4edb6795a1f4b299ab2a6f492bbfa17d9bbe2eee8922d949528a80ed30fR50

Looks like a missed update.

Reply:
I think the usage is something like 
```go
clusterClient := NewMonitoredClusterClient(name, option)
MonitorPoolStats(ctx, clusterClient, name ...)
```